slugid.js

```diff
@@ -20,16 +20,15 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-var uuidv4 = require('uuid/v4');
-var uuidParse = require('uuid-parse');
+var uuid = require('uuid');
```

Comment:
Nit: this could be:
```suggestion
const uuid = require('uuid');
```

It may be better to change all the imports in the future as part of converting to ``import`` syntax.

Reply:
Seems a bit of a cherry-picked sore spot. There's plenty of minor improvements like this that could be made. Such as general `var` usage or the import/module pattern.

I'm happy to make the change if that helps get this PR merged. Personally, I wouldn't make the change to `const` without a proceeding `"use strict";` statement.

Reply:
For me, "nit" means it is a suggestion, often based on coding style, that the submitter can take or leave, and doesn't need to be changed for this PR to merge. I think it is OK to keep this change focused on the ``uuid`` upgrade, rather than chase down all the issues, and that it will be cleaner due to that focus.

The reason I didn't merge immediately is that I wanted to get some further feedback from Mozilla team members with more Taskcluster experience. I've gotten a little in other channels, and none that suggests changes or blocks merging. I'm happy to merge in a few hours.

> Personally, I wouldn't make the change to const without a proceeding "use strict"; statement. The travis-ci config claims to test against nodejs v4.

I think this module has gone a bit stale, since changes were unneeded. I think we should add current node versions to the support matrix, and consider dropping old node versions, as part of the next release. That's definitely outside the scope of this PR.